irc/bitlbee/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                bitlbee
-version             3.5.1
-revision            1
+version             3.6.1
+revision            0
 categories          irc net
 platforms           darwin
```

Comment:
this is the default value for `platforms` since MacPorts v2.7 so the line can/should be removed